zerver/lib/actions.py

```diff
@@ -5733,8 +5733,7 @@ def do_delete_messages(realm: Realm, messages: Iterable[Message]) -> None:
         subscribers = get_active_subscriptions_for_stream_id(stream_id)
         # We exclude long-term idle users, since they by definition have no active clients.
         subscribers = subscribers.exclude(user_profile__long_term_idle=True)
-        subscriber_ids = [user.user_profile_id for user in subscribers]
-        users_to_notify = list(map(subscriber_info, subscriber_ids))
+        users_to_notify = list(subscribers.values_list("user_profile_id", flat=True))
```

Comment:
Can you explain this change with more of an example?

Reply:
@timabbott In 3922fb3a92cd6ee21ae2637bd3a6533a3a9d7339 and 2d544250b77b21a39028c06825fa19ab60489974 we change this function to send a list of user ids rather than dicts. https://github.com/zulip/zulip/pull/13964#discussion_r381585714 is the PR comment motivating that old change iirc.

Reply:
OK, yeah, this makes sense -- the key thing is that we're actually already using this format in the `do_update_message` code path.

I also added 48d99886fd409adf5ae448a01c4d25ca653cc3b2 on top.